### PR TITLE
Azure-arm-rest v236

### PR DESCRIPTION
**Pull request by LeftTwixWand**
**Task names**:
- AzureFileCopyV2
- AzureFileCopyV3
- AzureFileCopyV4
- AzureFileCopyV5
- AzureKeyVaultV1
- AzureKeyVaultV2
- AzureRmWebAppDeploymentV3
- AzureRmWebAppDeploymentV4

**Description**: I updated `azure-pipelines-tasks-azure-arm-rest` npm package to version `3.236.0` in all the dependent tasks.

**Documentation changes required:**  No

**Added unit tests:** No

**Attached related issue:** https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2108744

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by onetocny — APPROVED** [review]

**Review by manolerazvan — APPROVED** [review]

**Review by v-mohithgc — CHANGES_REQUESTED** [review]
changes under *AzureFileCopyV5\_buildConfigs\Node20 are not reflected after the build, please delete those files and run the build command again, hopefully that might fix the CI check failure. 
image

File: _generated/AzurePowerShellV4/task.json [hunk 1]
```diff
@@ -211,7 +211,7 @@
     "ExpiredServicePrincipal": "Could not fetch access token for Azure. Verify if the Service Principal used is valid and not expired."
   },
   "_buildConfigMapping": {
-    "Default": "4.236.0",
-    "Node20_229_2": "4.236.1"
+    "Default": "4.237.0",
```
Comment (v-mohithgc) [inline _generated/AzurePowerShellV4/task.json:214]: Ig these files(azurepowershell related) shouldn't be part of this PR.

**Review by v-mohithgc — CHANGES_REQUESTED** [review]

**Review by FinVamp1 — CHANGES_REQUESTED** [review]
For the Azure Function Apps Tasks I had already merged a PR that added new Functionality and revved  "azure-pipelines-tasks-azure-arm-rest": "^3.236.0",  Can you please update this branch with those changes? Then we don't need to increment the patch number I think.
